trappy/ftrace.py

```diff
@@ -85,6 +133,10 @@ def __init__(self, name="", normalize_time=True, scope="all",
         if normalize_time:
             self.normalize_time()
 
+        _create_trace_cache(self.trace_path)
+        for trace_class in self.trace_classes:
+            trace_class.write_csv(cache_path + '/' + trace_class.unique_word[:-1] + '.csv')
```

Comment:
Actually, I didn't agree with the last point why this should be a try/except? If we fail to write to the cache for some reason, I'd rather treat it as a failure and consider it a bug. Are you worried about the system running out of space?